zombies.js

```diff
@@ -241,20 +339,26 @@
  * @param {number} speed            The zombie's speed.
  */
 
+function FastZombie(health, strength, speed) {
+  Zombie.call(this, health, strength, speed);
+}
+FastZombie.prototype = Object.create(Zombie.prototype, {
+  constructor: {
+    value: Zombie
+  }
+});
```

Comment:
This should follow structure: `DeathMetalSong.prototype = Object.create(MetalSong.prototype);`